docs/it/metadata_oidc_op_cie.rst

```diff
@@ -0,0 +1,261 @@
+     - |uncheck-icon|
+
+
+Per favorire la minimizzazione dei dati, al posto di condividere un insieme fisso di **Claim Utente** con gli **RP**, 
+i Claim Utente necessari possono essere richiesti dall'**RP** stesso nell'**Authorization Request** usando lo scope del
+claim **claims**.
+
+Sulla base dello scope scelt, l'OP restituisce un insieme predefinito di Claim Utente CIE OIDC definisce i 
+seguenti valori dello **scope** da usare per richiedere i Claim:
+
+.. list-table:: 
+   :widths: 20 60 20
+   :header-rows: 1
+
+   * - **Claim**
+     - **Descrizione**
+     - **Obbligatorio**
+   * - **profile**
+     - Questo valore richiede l'accesso ai Claim di default del profilo dell'utente finale, 
+       che nello scenario CIE OIDC corrisponde al minimo dataset di eIDAS (es. cognome, nome, codice fiscale, data di nascita)
+     - |uncheck-icon|
+   * - **email**
+     - Questo valore richiede laccesso all'**email** e **email_verified**
+     - |uncheck-icon|
+
+.. note::
+   
+   Usando lo scope **openid** non si ottengono Claim Utente, se non specificati nel parametro claims. 
+   Solo il valore **sub** viene incluso.
+   Si POSSONO usare valori multipli dello scope, creando una lista "case sensitive" di valori ASCII delimitati da spazi. 
+   Claim Utente individuali possono essere richiesti usando il parametro di richiesta **claims** (vedere :ref:`Parametro Claims <claims_parameter>`). 
+   Usare il parametro **claims** è l'unico modo per richiedere specifiche combinazioni di Claim Utente che non possono
+   essere specificati usando valori di scope. 
+   Il parametro OIDC *claims* definisce un meccanismo per richiedere esplicitamente i Claim Utente da restituire
+   dall'**UserInfo Endpoint** e/o nell'**ID Token**. 
+   I membri top-level dell'Oggetto JSON della richiesta **claims** sono:
+
+
+.. list-table:: 
+   :widths: 20 60 20
+   :header-rows: 1
+
+   * - **Claim**
+     - **Descrizione**
+     - **Obbligatorio**
+   * - **userinfo**
+     - Richiede che i claim della lista siano ritornati dall'**UserInfo Endpoint**. 
+       Se presenti, i claim della lista vengono aggiunti agli altri claim richiesti usando valori scope. 
+       Se non sono presenti, i claim restituiti dall'UserInfo Endpoint saranno solo quelli richiesti usando i valori scope.
+     - |uncheck-icon|
+   * - **id_token**
+     - Richiede che i claim della lista siano ritornati nell'**ID Token**. 
+       Se presenti, i claim della lista vengono aggiunti ai claim di default dell'ID Token. 
+       Se non sono presenti, vengono richiesti i claim di default dell'ID Token.
+     - |uncheck-icon|
+
+Nello scenario CIE OIDC l'RP può solo chiedere **given_name**, **family_name**, **birthdate** e
+**fiscal_number** nell'**id_token object** del parametro **claim**.
+Gli altri claim riportati DEVONO essere richiesti nell'oggetto **userinfo** del parametro **claim**.
+Se viene omessa la richiesta **claims** e lo scope è uguale a **openid**, l'OP DOVREBBE fornire un insieme
```

Comment:
In the CIE OIDC scenario: 

- The RP is only allowed to ask for given_name, family_name, birthdate, and https://attributes.eid.gov.it/fiscal_number in the id_token object of claim parameter and the other reported claims MUST be requested in the userinfo object of claim parameter. 

- If the claims request is omitted and scope is equal to openid, the OP SHOULD provide only the pairwise sub claim both in the userinfo response and in the ID Token. For privacy reasons, the needed claims SHOULD be explicitly requested by the RP in the authorization request.

- The claims requested in the scope parameters are returned both in the userinfo response and in the ID Token

@sp640919 , potresti allineare? 
@peppelinux

Reply:
riporto testo: "L'RP può solo chiedere **given_name**, **family_name**, **birthdate** e **fiscal_number** 
nell'**id_token object** del parametro **claim**. Gli altri claim riportati DEVONO essere richiesti nell'oggetto **userinfo** del parametro **claim**.

Se viene omessa la richiesta **claims** e lo scope è uguale a **openid**, l'OP DOVREBBE fornire solo
il claim **sub** valorizzato a **pairwise**.
Pe motivi di privacy, i claim necessari DOVREBBERO essere richiesti esplicitamente dall'RP nell'Authorization Request.

I claim richiesti nei parametri di scope vengono ritornati sia nella risposta userinfo, che nell'ID Token. "